database/migrations/2023_09_29_132401_create_category_products.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    public function up(): void
+    {
+        Schema::create('category_product', function (Blueprint $table) {
+            $table->id();
+            $table->foreignUuid('product_id')->constrained();
+            $table->foreignId('category_id')->constrained();
+            $table->timestamps();   
+        });
+    }
```

Comment:
give migration pr only and make branch from it for product & category attach functionality.